src/jimgw/core/single_event/data.py

```diff
@@ -330,6 +331,38 @@ def from_fd(
         ), "Frequencies do not match after slicing"
         return data
 
+    @classmethod
```

Comment:
Save and load data from a file. I don't think this would impact the original library performance